src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
 {
+    using System.Linq;
```

Comment:
I believe our style guidelines specify that all `using`s must go outside the namespace.